spec.md

```diff
@@ -566,7 +566,6 @@ The descriptors MUST include an `artifactType` field that is set to the value of
 If the `artifactType` is empty or missing in the image manifest, the value of `artifactType` MUST be set to the config descriptor `mediaType` value.
 The descriptors MUST include annotations from the image manifest.
 If a query results in no matching referrers, an empty manifest list MUST be returned.
-If a manifest with the digest `<digest>` does not exist, a registry MAY return an empty manifest list.
 After a manifest with the digest `<digest>` is pushed, the registry MUST include previously pushed entries in the referrers list.
```

Comment:
not following the logic..
if no matching referrers for the digest.. MUST return empty results
.... now proposing removal of the MAY return empty results if the digest is not present...
Then next rule on 570 is **After** the digest is pushed, the registry MUST include the previously pushed matching referrers

To be consistent with your removal of line 569, you'd also have to remove the first clause of 570 (the after clause)

Reply:
Good catch on missing line 570, fixed.